accelerator/http.c

```diff
@@ -99,6 +99,17 @@ static inline int process_generate_address_request(ta_http_t *const http,
   return set_response_content(ret, out);
 }
 
+static inline int process_find_transactions_by_tag_request(
```

Comment:
The name scheme `process_find_transactions_by_tag_request` is quite longer than I expected. Can you shorten it?